dotnet/.config/dotnet-tools.json

```diff
@@ -6,13 +6,22 @@
       "version": "0.1.205",
       "commands": [
         "dotnet-repl"
-      ]
+      ],
+      "rollForward": true
     },
     "docfx": {
       "version": "2.67.5",
       "commands": [
         "docfx"
-      ]
+      ],
+      "rollForward": true
+    },
+    "microsoft.autogen.agenthost": {
```

Comment:
I believe this gets added when you created the local install of agenthost via:

```
dotnet tool install --add-source ./output/release Microsoft.AutoGen.AgentHost
```

Not sure if we want to have it installed by default, since it may not be possible until we do the release. Especially in the 0.4.0 (without `-dev.x` suffix)

Reply:
removed

Reply:
removed